xlators/mgmt/glusterd/src/glusterd-utils.c

```diff
@@ -5265,7 +5270,7 @@ glusterd_import_friend_volumes_synctask(void *opaque)
     conf->restart_bricks = _gf_true;
 
     while (i <= count) {
```

Comment:
If `count` is very big, we should avoid checking for each single volume. We should take advantage of `arg->status_arr` and efficiently skip entries that are nor modified.